src/nft/components/collection/Activity.tsx

```diff
@@ -138,9 +137,11 @@ export const Activity = ({ contractAddress, rarityVerified, collectionName, chai
                 (event, i) =>
                   event.eventType && (
                     <Box
```

Comment:
we'll need to update the styling to get rid of the link underlines introduced by this change https://interface-git-router-new-uniswap.vercel.app/nfts/collection/0xbc4ca0eda7647a8ab7c2061c2e118a18a936f13d/activity
![Screenshot 2023-08-01 at 11 15 46 AM](https://github.com/Uniswap/interface/assets/11512321/1e3d4901-67b1-49c9-91b0-f1b5e9a648e1)

Reply:
compared to staging https://app.corn-staging.com/#/nfts/collection/0xbc4ca0eda7647a8ab7c2061c2e118a18a936f13d/activity
![Screenshot 2023-08-01 at 11 17 08 AM](https://github.com/Uniswap/interface/assets/11512321/7b4fa3d5-1216-496f-9a63-0b4aeeb8d9d6)

Reply:
Good catch. I'll take a look at that.

Reply:
Addressed in a follow-up comment